src/Compilers/CSharp/Portable/Syntax/LocalFunctionStatementSyntax.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.CodeAnalysis.CSharp
+{
+    public partial class SyntaxFactory
+    {
+        public static LocalFunctionStatementSyntax LocalFunctionStatement(
```

Comment:
consider doc'ing this is for binary compat.

Reply:
Do you mean a `//` doc for the maintainers of this file?